notebooks/executing-notebooks-from-another-notebook-with-fusion-sql/meta.toml

```diff
@@ -0,0 +1,13 @@
+[meta]
+authors=["singlestore"]
+title="Executing Notebooks from Another Notebook with Fusion SQL"
```

Comment:
Maybe rename all occurrences of `execute/executing` to `run/running` for consistency with the actions in the UI and `%run` magic command (including folder name).